src/agent/Cargo.toml

```diff
@@ -73,14 +73,12 @@ clap = { version = "3.0.1", features = ["derive"] }
 strum = "0.26.2"
 strum_macros = "0.26.2"
 
-# Communication with the OPA service
-http = { version = "0.2.8", optional = true }
-reqwest = { version = "0.11.14", optional = true }
-# The "vendored" feature for openssl is required for musl build
-openssl = { version = "0.10.54", features = ["vendored"], optional = true }
-
 # Image pull/decrypt
 image-rs = { git = "https://github.com/confidential-containers/guest-components", rev = "ca6b438", default-features = true, optional = true }
+openssl = { version = "0.10.54", features = ["vendored"], optional = true }
+
+# Agent Policy
+regorus = { git = "https://github.com/danmihai1/regorus", rev = "a615f40", default-features = false, features = ["arc", "regex"], optional = true }
```

Comment:
It is better to fetch regorus from crates.io once I push a release that meets this PR's requirements. Similar to what `trustee` does
https://github.com/confidential-containers/trustee/blob/dc01f454264fb4350e5f69eba05683a9a1882c41/Cargo.toml#L37

Reply:
@anakrish , CI tests are now passing using:

git = "https://github.com/microsoft/regorus", rev = "316f3a76"

So, I think this would be a good time to publish that into crates.io.